### DISCUSSION TITLE
Configuration

### DISCUSSION THREAD

**Discussion by fsarra-dev** [primefaces/community#2096] [PrimeFaces] [answered]
Hello there I'm a starter in development world. I'm working in a jsf and got issue with configuration with primefaces. I'm using jar to learn. Java 21 and jar v 12.0.0 primefaces. The browser doesn't understand the tag "p:" so I can't use the power UI of primefaces. There is a configuration about this topic? Like step by step? Thank you all.

**Comment by tandraschko** [comment, marked as answer]
You need to setup a valid JakartaEE project first and chooce a application server, where you can run/deploy it.

PrimeFaces is just a component library for the JakartaEE stack.


Here is a example for Payara: https://blog.payara.fish/getting-started-with-jakarta-ee-9-jakarta-faces-jsf

this is not really PrimeFaces releated.

**Reply by fsarra-dev** [reply]
Thank you for the reply.
I'm using Eclipse and Wildfly30. I have inserted jar of Primefaces in lib, but tag "p:" stil doesn't rendering on edje browser, while "h:" is rendering, maybe it's the import up of xhtml file?

I tried this one from primefaces

<html xmlns="http://www.w3.org/1999/xhtml"

    xmlns:h="http://java.sun.com/jsf/html"

    xmlns:f="http://java.sun.com/jsf/core"

    xmlns:p="http://primefaces.org/ui">

 

     <h:head>

 

     </h:head>

 

     <h:body>

 

         <p:button value="test" />

 

     </h:body>

</html>


Deploy well, but button does not appear on Dom.
